packages/react-monaco-editor/scripts/copyTypes.ts

```diff
@@ -15,18 +14,18 @@ export function copyTypes() {
     const pkg = packagesToResolve.shift();
     resolvedPackages.push(pkg);
 
-    const packageMatch = pkg.match(/^(@uifabric\/|@fluentui\/)?([\w-]+)/);
-    const dtsPath = expandSourcePath(`${pkg}/dist/${packageMatch[2]}.d.ts`);
+    const packageMatch = pkg.match(/^@fluentui\/([\w-]+)/);
+    const dtsPath = expandSourcePath(`${pkg}/dist/${packageMatch[1]}.d.ts`);
 
     if (fs.existsSync(dtsPath)) {
       // copy this .d.ts
       pathsToCopy.push(dtsPath);
 
-      // add any other @uifabric or @fluentui packages it references for processing
+      // add any other @fluentui packages it references for processing
       // (ignore React imports and other imports)
       const dtsContents = fs.readFileSync(dtsPath).toString();
-      const importRegex = /(?:import|export) .*? from ['"](@(?:uifabric|fluentui)\/[\w-]+)/gm;
-      let importMatch;
+      const importRegex = /(?:import|export) .*? from ['"](@fluentui\/[\w-]+)/gm;
```

Comment:
previously this as using non capture group for the module import, this breaks that behaviour. I don't see any test that can confirm this works as expected. is this intended ?

Reply:
The outer capturing group 1 is still the same, I just removed an inner non-capturing group. I tested it and it works. (Setting up infrastructure to add tests for this file is *very* much out of scope at this point.) 
<img width="329" alt="image" src="https://user-images.githubusercontent.com/5864305/162828107-92de75d1-09ea-41ae-86de-4ed713a3a9da.png">